routes/o2auth_google.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express');
+const { OAuth2Client } = require('google-auth-library');
+const router = express.Router();
+const User = require('../models/user'); 
+const jwt = require('jsonwebtoken');
+
+// Initialize Google OAuth client
+const googleClient = new OAuth2Client('243962731858-rb04b0dbelevp5blhc339io995opuhqu.apps.googleusercontent.com');
```

Comment:
Are you sure we can add this in public repository? Can you create an environment variable for the key?